### PR TITLE
Implement device await command

### PR DESCRIPTION
**Pull request by jviotti**

### CODE REVIEW

File: lib/actions/device.coffee [hunk 1]
```diff
@@ -163,6 +163,46 @@ exports.supported =
...
 
+exports.await =
+	signature: 'device await <name>'
+	description: 'await for a device to become online'
+	help: '''
+		Use this command to await for a device to become online.
+
+		The process will exit when the device becomes online.
+
+		Notice that there is not time limit for this command, so it might run forever.
```
Comment (emirotin) [inline lib/actions/device.coffee]: "no time limit"

File: lib/actions/device.coffee [hunk 2]
```diff
@@ -163,6 +163,46 @@ exports.supported =
...
+
+		You can configure the poll timeout with the --interval option (defaults to 3000ms).
```
Comment (emirotin) [inline lib/actions/device.coffee]: better to call it interval, not timeout

**Comment by emirotin** [comment]
lgtm except of the small comments above

File: lib/actions/device.coffee (hunk 1 above)
Comment (jviotti) [inline lib/actions/device.coffee]: Nice catch, fixed.

File: lib/actions/device.coffee (hunk 2 above)
Comment (jviotti) [inline lib/actions/device.coffee]: Fixed.

**Pull request merged**